packages/app-bridge/src/react/useDocumentPageTargets.spec.ts

```diff
@@ -0,0 +1,35 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { act, renderHook } from '@testing-library/react';
+import { afterEach, describe, expect, it, vi } from 'vitest';
+
+import type { AppBridgeTheme } from '../AppBridgeTheme';
+
+import { TargetsDummy } from '../tests';
+import { useDocumentPageTargets } from './useDocumentPageTargets';
+
+const DOCUMENT_ID = 1;
```

Comment:
```suggestion
const DOCUMENT_PAGE_ID = 345;
```
And let change 1 to another value which you can ensure is not just chance